python/foglamp/services/core/api/snapshot_config.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+
+from aiohttp import web
+from foglamp.services.core.connect import *
+
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_help = """
+    -------------------------------------------------------------------------------
+    | GET POST        | /foglamp/category/snapshot                                |
+    | PUT DELETE      | /foglamp/category/snapshot/{id}                           |
+    -------------------------------------------------------------------------------
+"""
+
+
+async def get_snapshot(request):
+    """ get list of available snapshots
+
+    :Example:
+        curl -X GET http://localhost:8081/foglamp/category/snapshot
+    """
+    _storage = get_storage_async()  # from foglamp.services.core.connect
+    try:
+        retval = await _storage.get_snapshot('configuration')
+    except Exception as ex:
```

Comment:
Please handle StorageServer Error, raised by storage client (for all endpoint def)